main.rb

```diff
@@ -0,0 +1,21 @@
+def bubble_sort(item)
+  (1..(item.length - 1)).each do
+    (0..(item.length - 2)).each do |k|
+      item[k], item[k + 1] = item[k + 1], item[k] if item[k].to_i > item[k + 1].to_i
+    end
+  end
+  item = item.inspect
+end
```

Comment:
Your code is returning the correct value but the wrong value type. For example, instead of returning an `array` -- `[1,2,3,4,5,6,7,8]`, it returns a string `"[1,2,3,4,5,6,7,8]"`

* [ ] bubble_sort method is implemented and working ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_bubble_sort/sort.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-1))

Reply:
In our last commit we fix this issue